xwiki-platform-core/xwiki-platform-like/xwiki-platform-like-api/src/main/resources/templates/likers.vm

```diff
@@ -118,5 +119,5 @@
         'translationPrefix' : "xe.admin.users."
     })
 
-    #livetable("likers" $columns $columnsProperties $options)
+    #livetable("likersTable" $columns $columnsProperties $options)
```

Comment:
Why do you need to change the live table id?

Reply:
Because it collides with the item in the #tmMoreAction menu.

Reply:
Without this change the livetable wouldn't work properly anymore when adding the contentheader

Reply:
ok, maybe add a comment explaining that the is a risk of "id collision".

Reply:
I wasn't sure such a comment would really be helpful on the long term / outside of the context of this PR (not really any reason to change this livetable ID again). 
I added a comment in https://github.com/xwiki/xwiki-platform/pull/2788/commits/95a0d0fb0037158ec197d0b1464f66a76d919595 . :+1: